### ISSUE TITLE
Guests should not be able to play ranked

### ISSUE THREAD

**Issue by ursi** [alcalyn/hex#39]
It makes it way to cheat the rankings. I also have had multiple guests accept my ranked games, and if I had to guess, are never going to make a move.

**Comment by alcalyn** [comment]
I agree that there is things to disallow to guests, like accepting members correspondence games, allow members to disallow guest join their games, prevent guest chatting in members games, but I want guests have almost all features as a member account.
As guest, playing ranked vs other guest should be allowed, then you can create an account to keep your rating. Creating an account actually "upgrade" your guest account, you keep the same account.
